### PR TITLE
fix changed path and run prettier

### PR DESCRIPTION
**Pull request by calcsam**
Missed this earlier with rebasing. `path` needs to reference `@arkw/core` instead of `core`

### CODE REVIEW

File: packages/cli/src/commands/init.ts [hunk 1]
```diff
@@ -49,8 +55,8 @@ async function migrate(createOnly = false) {
       {
         env: {
           ...process.env,
-          FUTURE_DATABASE_URL: "postgresql://postgres:postgres@127.0.0.1:54322/arkwright?schema=public",
-          DB_URL: "postgresql://postgres:postgres@127.0.0.1:54322/arkwright?schema=public",
+          FUTURE_DATABASE_URL: 'postgresql://postgres:postgres@127.0.0.1:54322/arkwright?schema=public',
```
Comment (abhiaiyer91) [inline packages/cli/src/commands/init.ts]: this var isnt used anymore can be removed

**Review by abhiaiyer91 — APPROVED** [review]

File: packages/cli/src/files/starter-config.ts [hunk 2]
```diff
@@ -5,10 +5,9 @@ export const config: Config = {
   //logConfig: {}, // TODO: Add this
   systemActions: [],
   systemEvents: [],
-  plugins: [
-  ],
+  plugins: [],
```
Comment (Joshuafolorunsho) [inline packages/cli/src/files/starter-config.ts]: We might want to update this to `integrations` to match the new naming convention.

**Review by Joshuafolorunsho — APPROVED** [review]
LGTM.

**Review by Joshuafolorunsho — APPROVED** [review]

File: packages/cli/src/files/starter-config.ts (hunk 2 above)
Comment (calcsam) [reply packages/cli/src/files/starter-config.ts]: awesome, addressed

File: packages/cli/src/commands/init.ts (hunk 1 above)
Comment (calcsam) [reply packages/cli/src/commands/init.ts]: great thx

**Pull request merged**